src/d3q27_viscoplastic/Dynamics.R

```diff
@@ -0,0 +1,86 @@
+Ave = FALSE
+x = c(0,1,-1);
+P = expand.grid(x=0:2,y=0:2,z=0:2)
+U = expand.grid(x,x,x)
+
+AddDensity(
+        name = paste("f",P$x,P$y,P$z,sep=""),
+        dx   = U[,1],
+        dy   = U[,2],
+        dz   = U[,3],
+        comment=paste("density F",1:27-1),
+        group="f"
+)
+
+# exported to VTK
+AddQuantity( name="P",unit="Pa")
+AddQuantity( name="U",unit="m/s",vector=T)
+AddQuantity( name="nu_app",unit="m2/s")
+AddQuantity( name="yield_stat")
+
+AddSetting(name="nu", default=0.16666666, comment='Viscosity')
+AddSetting(name="Velocity", default="0m/s", comment='Inlet velocity', zonal=TRUE)
+AddSetting(name="Pressure", default="0Pa", comment='Inlet pressure', zonal=TRUE)
+	
+AddDensity(name="nu_app", dx=0, dy=0, dz=0)
+AddDensity(name="yield_stat", dx=0, dy=0, dz=0)
+
+AddSetting(name="GalileanCorrection",default=1.,comment='Galilean correction term')
+AddSetting(name="ForceX",default=0, comment='Force force X')
+AddSetting(name="ForceY",default=0, comment='Force force Y')
+AddSetting(name="ForceZ",default=0, comment='Force force Z')
+AddSetting(name="SigmaY",default=0, comment='Yield stress')
+
+AddGlobal(name="Flux", comment='Volume flux', unit="m3/s")
+
+AddNodeType("SymmetryY", "BOUNDARY")
+AddNodeType("SymmetryZ", "BOUNDARY")
+AddNodeType("TopSymmetry","BOUNDARY")
+AddNodeType("BottomSymmetry","BOUNDARY")
+AddNodeType("NVelocity", "BOUNDARY")
+AddNodeType("SVelocity", "BOUNDARY")
+AddNodeType("NPressure", "BOUNDARY")
+AddNodeType("SPressure", "BOUNDARY")
+
+# new boundary conditions
+AddNodeType("CoutVelocity","BOUNDARY")
+AddNodeType("PCoutVelocity","BOUNDARY")
+AddNodeType("WVelocityEq","BOUNDARY")
+AddNodeType("WVelocityBB","BOUNDARY")
```

Comment:
actually, there are no options for "WVelocityEq" and "WVelocityBB" -will be removed. Naming convention will be followed